app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
I thought we talked way back about the root division and decided that we needed one. We need to be able to run reports on all divisions in the system. Having a root division seems like the obvious way to do that within the permission system we've designed. This root division stuff is turning out to be pretty complicated, so maybe it would have been simpler to add a super admin flag to some users. But if the code is already written that way, and refactoring would be complicated, it doesn't seem worthwhile to refactor it now. We have a budget here and we have used about 3/4 of the contingency already. 

That said, I don't understand why the code is relying so heavily on there being a root division. I thought the idea was that we would just make one, and then its members would be effective super admins. And I'd love to avoid specifying primary keys in the code. It doesn't seem like that should be necessary. 

I don't think you should use `assert` in a model. Raise an error if you need to. 

Seeds should contain things that the system will break without, right? So shouldn't that include at least one loan status (active)?